### ISSUE TITLE
Paying entirely with store credit should remove the need to supply a credit card.

### ISSUE THREAD

**Issue by mleglise** [spree-contrib/spree_store_credit_payment_method#25]
Currently, the user is still prompted for a credit card, and cannot complete checkout without it. The Bonobos frontend appears to have already fixed this.

Is there a suggested way to fix this in other frontends?

**Comment by jasonfb** [comment]
we took care of this in our controller code

**Comment by JDutil** [comment]
@jasonfb if your up for it a PR against the store credits branch would be nice https://github.com/spree/spree/pull/5917

**Comment by jasonfb** [comment]
Yes I was wondering about that. I assume that pull is targeted against 3.1?

**Comment by JDutil** [comment]
Yea targeted on master branch to be included whenever it's finished.  I'm pretty sure it just needs the views, and to make sure specs are passing I could be wrong though need to have @peterberkenbosch add a task list for whats left at some point.

**Comment by dbruns** [comment]
@jasonfb What was the actual fix for this?
